directgov_passportapplication.yml

```diff
@@ -0,0 +1,7 @@
+---
+site: directgov_passportapplication
+whitehall_slug: hm-passport-office
+homepage: https://passportapplication.service.gov.uk
+tna_timestamp: 20201010101010 # Stub timestamp - site not in TNA
+host: passportapplication.direct.gov.uk
+global: =301 https://passportapplication.service.gov.uk
```

Comment:
Should we redirect to https://passportapplication.service.gov.uk/ips-olc/ instead? https://passportapplication.service.gov.uk redirects to there, so it would avoid a second hop.